templates/taxi/driver_list.html

```diff
@@ -8,6 +8,7 @@ <h1>Driver list</h1>
         <li>
           <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.id == user.id %}(Me){% endif %}
```

Comment:
implement it via a table  instead of list

Reply:
code refactoring? 
I understand correctly?